### DISCUSSION TITLE
Stream DLS and MOT info

### DISCUSSION THREAD

**Discussion by Apfelsafft** [AlbrechtL/welle.io#838] [Q&A] [answered]
Hello,

I am using Music Assistant to play music and I managed to listen to DAB+ radio streaming a certain station as mp3 using welle-cli.
Now I wonder if it also possible to stream the DLS and MOT info to the player like internet radio stations do?

As I am quite new to DAB and SDR I am pretty sure I am missing something.

Btw, awesome that there is a new version right before christmas!
Thanks and enjoy the upcoming days.

Best,
René

**Comment by AlbrechtL** [comment, marked as answer]
Hello @Apfelsafft,

Great use case to listen DAB in Home Assistant :-) and nice to hear that you connected Music Assistant and welle-cli together. Regarding your question: Currently, DLS and the MOT images are not included in the MP3 stream. I think is it possible to include the DLS text in the MP3 stream, but somebody has to implement it. Regarding the MOT images, I'm not so optimistic. But I didn't check it in detail.

Pull request are welcome!

**Reply by Apfelsafft** [reply]
Hello @AlbrechtL,
good to hear that it is possible - somehow.
Unfortunately, I am not a capable developer to implement it in a perfect way.
But maybe I find some snippets around to put things together.

However, is there something like a roadmap on how you want to evolve welle.io and its components?

Best regards,
René

**Reply by AlbrechtL** [reply]
Since welle.io is only run by volunteers, there is no roadmap, unfortunately. But I have in my mind to add FM or [HD Radio](https://en.wikipedia.org/wiki/HD_Radio). But I personally use only DAB+ for that reason, my priority is pretty low to add FM or HD Radio.
